htdocs/societe/paymentmodes.php

```diff
@@ -1328,11 +1330,19 @@
 			foreach ($balance->pending as $cpt)
 			{
 		$arrayzerounitcurrency=array('BIF', 'CLP', 'DJF', 'GNF', 'JPY', 'KMF', 'KRW', 'MGA', 'PYG', 'RWF', 'VND', 'VUV', 'XAF', 'XOF', 'XPF');
-		if (! in_array($cpt->currency, $arrayzerounitcurrency)) $amount = $cpt->amount / 100;
-		else $amount = $cpt->amount;
-      print '<tr><td>'.$langs->trans("Pending").'</td><td>'.price($amount, 0, '', 1, - 1, - 1, strtoupper($cpt->currency)).' </td><td>'.$langs->trans("Currency".strtoupper($cpt->currency)).'</td></tr>';
+		if (! in_array($cpt->currency, $arrayzerounitcurrency))  $currencybalance[$cpt->currency]->pending=$currencybalance[$cpt->currency]->available+$cpt->amount / 100;
```

Comment:
This line looks strange: Don't you mean $amount instead of ($cpt->amount / 100) ?

Reply:
after test, pending value on stripe account is the difference between available and not just the not available value

i revamp for better presentation with 1 line by currency